packages/nvidia_nat_rag/src/nat/plugins/rag/client.py

```
@@ -0,0 +1,306 @@
+                try:
+                    parsed: ChainResponse = ChainResponse.model_validate_json(raw_chunk)
+                    if parsed.choices:
+                        choice = parsed.choices[0]
+                        if choice.delta and choice.delta.content:
+                            content = choice.delta.content
+                            if isinstance(content, str):
+                                chunks.append(content)
+                    if parsed.citations and parsed.citations.results:
+                        final_citations = parsed.citations
+                except Exception:
+                    continue
```

Ruff (0.14.13)

`try`-`except`-`continue` detected, consider logging the exception

(S112)